curriculum/challenges/english/25-front-end-development/quiz-bash-and-sql/66f1affc0ef4fcca423d4688.md

```diff
@@ -21,435 +21,509 @@ Placeholder question
 
```

Comment:
there is still written "Placeholder question" above this line, this should be the question